app/controllers/concerns/web_app_controller_concern.rb

```diff
@@ -11,7 +11,7 @@ module WebAppControllerConcern
   end
 
   def skip_csrf_meta_tags?
-    current_user.nil?
+    !(ENV["OMNIAUTH_ONLY"] == "true" && Devise.omniauth_providers.length == 1) |&& current_user.nil?
```

Comment:
There's a syntax error here.

```suggestion
    !(ENV["OMNIAUTH_ONLY"] == "true" && Devise.omniauth_providers.length == 1) && current_user.nil?
```